api/src/main/java/org/pmiops/workbench/cloudtasks/ForwardingCloudTasksStub.java

```diff
@@ -63,7 +65,9 @@ public ForwardingCloudTasksStub(String baseUrl) {
             String.format(
                 "asynchronously forwarding task request for queue '%s', to handler '%s'",
                 queueName.getQueue(), apiReq.url()));
-        new OkHttpClient()
+        OkHttpClient client = new OkHttpClient();
+        client.setReadTimeout(10, TimeUnit.MINUTES);
```

Comment:
Just checking, in addition to setting ReadTimeout, what about `connectTimeout`? The default value is 10 seconds which might not be enough considering how slow terra api services are.